internal/impl/snowflake/output_snowflake_streaming.go

```diff
@@ -22,18 +21,19 @@ import (
 )
 
 const (
-	ssoFieldAccount       = "account"
-	ssoFieldUser          = "user"
-	ssoFieldRole          = "role"
-	ssoFieldDB            = "database"
-	ssoFieldSchema        = "schema"
-	ssoFieldTable         = "table"
-	ssoFieldKey           = "private_key"
-	ssoFieldKeyFile       = "private_key_file"
-	ssoFieldKeyPass       = "private_key_pass"
-	ssoFieldBatching      = "batching"
-	ssoFieldChannelPrefix = "channel_prefix"
-	ssoFieldMapping       = "mapping"
+	ssoFieldAccount        = "account"
+	ssoFieldUser           = "user"
+	ssoFieldRole           = "role"
+	ssoFieldDB             = "database"
+	ssoFieldSchema         = "schema"
+	ssoFieldTable          = "table"
+	ssoFieldKey            = "private_key"
+	ssoFieldKeyFile        = "private_key_file"
+	ssoFieldKeyPass        = "private_key_pass"
+	ssoFieldInitStatements = "init_statements"
```

Comment:
Nit: I wonder if this might be a bit confusing for end users since the `sql_*` components have [`init_statement`](https://docs.redpanda.com/redpanda-connect/components/inputs/sql_select/#init_statement). For some drivers, you can use `;` to shove multiple statements in there.

Reply:
Do you think I should remove the plural from statements? I guess I can just update the description and examples to show multiple statements work

Reply:
That's what I was thinking since it's the same functionality, but up to you really.